LEGO1/mxeventmanager.cpp

```diff
@@ -1,13 +1,21 @@
 #include "mxeventmanager.h"
 
-// OFFSET: LEGO1 0x100c0360 STUB
+// OFFSET: LEGO1 0x100c0360
 MxEventManager::MxEventManager()
 {
-  // TODO
+  Init();
 }
 
-// OFFSET: LEGO1 0x100c03f0 STUB
+// OFFSET: LEGO1 0x100c03f0
 MxEventManager::~MxEventManager()
 {
-  // TODO
+  // TODO: MxMediaManager::TerminateThread call
 }
+
+// OFFSET: LEGO1 0x100c0450
+void MxEventManager::Init()
+{
+  // huh?
+  // imagine if the devs actually put this here to troll us
+  return;
```

Comment:
This probably had something during development but was later removed. The comments and `return` aren't necessary.